docs/bosh-troubleshooting-gcp.adoc

```diff
@@ -0,0 +1,59 @@
+= BOSH Troubleshooting
```

Comment:
ok, i was thrown off by this title.  in my mind, the intent here was not to troubleshoot anything, but to give the student a chance to validate their installation.  that is, by setting up their bosh target etc.. they confirm that the director has been created and is responsive/ready for subsequent deployments to the iaas.

so the basic idea here is to pull out this sort of minilab into its own document.  i wanted to make sure that it was worked on after the bosh director got installed.